Cannot launch Java Admin Client (JNLP)

### What is the problem
> I cannot launch the Java Admin Client. Upon launching exist.jnlp I get:

```
java.lang.NoClassDefFoundError: org/xmldb/api/base/XMLDBException
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
	at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
	at java.lang.Class.getMethod0(Class.java:3018)
	at java.lang.Class.getMethod(Class.java:1784)
	at com.sun.javaws.Launcher.doLaunchApp(Unknown Source)
	at com.sun.javaws.Launcher.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: org.xmldb.api.base.XMLDBException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at com.sun.jnlp.JNLPClassLoader.findClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at com.sun.jnlp.JNLPClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 8 more
```

If I do curl -v http://localhost:8080/xmlrpc I get:

```
* About to connect() to localhost port 8080 (#0)
*   Trying ::1...
* Connected to localhost (::1) port 8080 (#0)
> GET /xmlrpc HTTP/1.1
> User-Agent: curl/7.29.0
> Host: localhost:8080
> Accept: */*
> 
< HTTP/1.1 405 HTTP method GET is not supported by this URL
< Date: Mon, 26 Nov 2018 12:28:16 GMT
< Cache-Control: must-revalidate,no-cache,no-store
< Content-Type: text/html;charset=iso-8859-1
< Content-Length: 393
< Server: Jetty(9.4.10.v20180503)
< 
<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
<title>Error 405 HTTP method GET is not supported by this URL</title>
</head>
<body><h2>HTTP ERROR 405</h2>
<p>Problem accessing /xmlrpc. Reason:
<pre>    HTTP method GET is not supported by this URL</pre></p><hr><a href="http://eclipse.org/jetty">Powered by Jetty:// 9.4.10.v20180503</a><hr/>

</body>
</html>
```
### Describe how to reproduce
```
git clone git://github.com/eXist-db/exist.git
cd exist
git checkout master
./build.sh
./build.sh -f build/scripts/jarsigner.xml

<Set name="contextPath">/</Set>
in tools/jetty/webapps/exist-webapp-context.xml

cd tools/yajsw/bin/
./installDaemonNoPriv.sh
./startDaemon.sh
```

connect to the dashboard - click on 'Java Admin Client' - download the jnlp, and double click on it.

### Context information
Please always add the following information
- eXist-db 4.5.0  / e29b4099c71094e25fae7d627975bf8e6b78658e
- build 1.8.0_152-b16
- CentoS 7
- 64 bit
- Clone via GitHub

--- a/src/org/exist/webstart/JnlpJarFiles.java
+++ b/src/org/exist/webstart/JnlpJarFiles.java
@@ -73,7 +73,7 @@ public class JnlpJarFiles {
             "rsyntaxtextarea-%latest%",
             "slf4j-api-%latest%",
             "ws-commons-util-%latest%",
-            "xmldb",
+            "xmldb-api-%latest%",
             "xmlrpc-client-%latest%",
             "xmlrpc-common-%latest%"
     };

